src/components/InventoryTable/EntityTable.js

```diff
@@ -142,6 +142,7 @@ const EntityTable = ({
                         ...actions && rows?.length > 0 && { actions }
                     } }
                     { ...tableProps }
+                    isStickyHeader
```

Comment:
Can you please.move it before `tableProps`? That way consumers can rewrite it if they don't like the sticky header.